src/utils/__tests__/actions.spec.js

```diff
@@ -39,10 +43,12 @@ describe('actions', () => {
         back: sinon.spy(),
         forward: sinon.spy(),
         getField: sinon.stub().resolves(fieldObject),
+        getFieldDescription: sinon.stub().resolves(fieldInfoObject),
         getVariableByName: sinon.stub().resolves(variableObject),
         lockAll: sinon.spy(),
         unlockAll: sinon.spy(),
         getBookmarkList: () => [{ qData: { title: 'findMyBookmark' }, qInfo: { qId: 'myBookmarkId' } }],
+        evaluate: () => '43850;43881',
```

Comment:
This is not a very good mock. The only alternative I see is to find out how engine does the conversion, seems like overkill to me...